examples/tutorials/advanced/Federated SMS Spam prediction/Federated SMS Spam prediction.ipynb

```diff
@@ -0,0 +1,545 @@
+{
```

Comment:
<p>&gt; In order to address this issue, there are two solutions: Federated Learning with Multi-Party Computation and Encrypted Computation.</p><p><br></p><p>Differential Privacy would be used to make sure the model doesn't give access to some private information.</p><p><br></p><p>Multi-Party Computation (one kind of Encrypted Computation) in return allows you do send the model privately so that the remote workers which have the data can't see the weights you're using</p>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>